docs/modules/clusters/partials/ucn-migrate-tip.adoc

```diff
@@ -1 +1 @@
-CAUTION: {ucd} has been deprecated and will be removed in the next major version. To continue deploying your user code after this time, {open-source-product-name} users can either upgrade to {enterprise-product-name}, or add their resources to the Hazelcast member class paths. Hazelcast recommends that {enterprise-product-name} users migrate their user code to use {ucn} for all purposes other than Jet stream processing. For further information on migrating from {ucd} to {ucn}, see xref:clusters:ucn-migrate-ucd.adoc[].
\ No newline at end of file
+CAUTION: {ucd} has been deprecated and will be removed in the next major version. To continue deploying your user code after this time, {open-source-product-name} users can either upgrade to {enterprise-product-name}, or add their resources to the Hazelcast member class paths. Hazelcast recommends that {enterprise-product-name} users migrate their user code to use {ucn}. For further information on migrating from {ucd} to {ucn}, see xref:clusters:ucn-migrate-ucd.adoc[].
```

Comment:
Jet has ability to update job definition without losing intermediate results via `cancelAndExportSnapshot` + starting compatible job definition using named snapshot. Will that work for UCD->UCN migration (AFAIR we do not have a test for it)? Also, will that work for job attached resources -> UCN migration?

Reply:
see https://docs.hazelcast.com/hazelcast/5.5/pipelines/job-update - should be updated if we allow changing UCN in that way

Reply:
Since UCN and UCD both just provide classloaders above Jet with access to resources, there should not be any incompatibility beyond the UCN resources needing to match the UCD resources; this is how we have approached migration for all data structures, however there is no specific testing in place for this